src/python_x509_pkcs11/__init__.py

```diff
@@ -2,3 +2,7 @@
 """
 
 __version__ = "0.9.0"
+
+from .lib import DEFAULT_KEY_TYPE, KEYTYPES, get_keytypes_enum
```

Comment:
Why have imports in __init__.py? I'm a python novice and thought init file should be close to empty.

Reply:
For making import less troublesome, instead of remember all different submodules, one can import most of the important (and with clear names) attributes from the main package. We can even lock it down with `__all__` but that is something I left for a future release.